pkg/otlp/metrics/metrics_remapping.go

```diff
@@ -42,32 +44,32 @@ func remapSystemMetrics(all pmetric.MetricSlice, m pmetric.Metric) {
 	}
 	switch name {
 	case "system.cpu.load_average.1m":
-		copyMetric(all, m, "system.load.1", 1)
+		copyMetricWithAttr(all, m, "system.load.1", 1, attributesMapping{})
```

Comment:
nit: I think we should make `attributesMapping{}` a package level constant to avoid possibly allocating a new one for each of these calls

Reply:
Good call, created `var emptyAttributesMapping = attributesMapping{} `